### PR TITLE
Completed login page

### PR DESCRIPTION
**Pull request by riyajain59**
Login page frontend and backend completed.

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -4,24 +4,56 @@ const express = require('express');
...
+// const Likes = require("./models/Likes.js");
+// const User = require("./models/User.js");
 
 app.get('/', async (req, res) => {
   res.send('Hello World')
 });
 
+app.use('/auth/', authRoutes)
+
+app.get('/getProfile', async (req, res) => {
```
Comment (coconut750750) [inline backend/index.js]: you can delete this example endpoint

File: backend/index.js [hunk 2]
```diff
@@ -4,24 +4,56 @@ const express = require('express');
...
 
 mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true, useUnifiedTopology: true });
 var db = mongoose.connection;
 
-const Profile = require("./models/profile.js");
-const Likes = require("./models/Likes.js");
-const User = require("./models/User.js");
+ const Profile = require("./models/profile.js");
+// const Likes = require("./models/Likes.js");
+// const User = require("./models/User.js");
```
Comment (coconut750750) [inline backend/index.js]: you can delete these three lines (lines 39-41)

File: backend/routes/auth.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            return next(err);
+        }   
+
+        res.sendStatus(200);
+    });
+    
+});
+
+router.post('/login', async (req, res, next) => {
+    console.log("hi im here");
```
Comment (coconut750750) [inline backend/routes/auth.js]: delete these 2 console.logs (line 22-23)

File: frontend/src/utils/api.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export async function callApi() {
```
Comment (coconut750750) [inline frontend/src/utils/api.js]: you can also delete this function

**Review by coconut750750 — CHANGES_REQUESTED** [review]

**Review by coconut750750 — APPROVED** [review]

File: backend/yarn.lock [hunk 5]
```diff
@@ -447,6 +474,12 @@ fresh@0.5.2:
   resolved "https://registry.yarnpkg.com/fresh/-/fresh-0.5.2.tgz#3d8cadd90d976569fa835ab1f8e4b23a105605a7"
   integrity sha1-PYyt2Q2XZWn6g1qx+OSyOhBWBac=
 
+<<<<<<< HEAD
```
Comment (alicesf2) [inline backend/yarn.lock]: make sure to delete these merge conflict tags

File: frontend/src/index.css [hunk 6]
```diff
@@ -11,3 +12,48 @@ code {
...
+}
+
+@media (min-width: 769px) {
+  .container {
+    margin: 2.5rem auto;
+  }
+}
+
+
+/* Quiz Component */
```
Comment (alicesf2) [inline frontend/src/index.css:42]: good use of css :)

File: frontend/src/pages/LoginPage.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        <div class="inputOne">
+        <input type="text" placeholder='Password' onChange={event => setPassword(event.target.value)} required/>
+      </div>
+      <h2><u>Forgot Password?</u></h2>
+      <button onClick = {() => login(username, password).then(data => {
+        if (data.status != 401) {
+          props.history.push('/feed');
+          
+        }
+        console.log(data);
```
Comment (alicesf2) [inline frontend/src/pages/LoginPage.js:39]: remove console logs

File: backend/yarn.lock (hunk 5 above)
Comment (coconut750750) [reply backend/yarn.lock]: oops, thats me

**Review by alicesf2 — APPROVED** [review]
Awesome job :riyawn:! Just resolve the merge conflicts in yarn.lock and you'll be set 👍

File: backend/yarn.lock (hunk 5 above)
Comment (alicesf2) [reply backend/yarn.lock]: wait brandon they're still there

**Pull request merged**